services/merak-topo/utils/utils.go

```diff
@@ -40,11 +57,15 @@ func K8sClient() (*kubernetes.Clientset, error) {
 
 func K8sConfig() (*rest.Config, error) {
 
+	var err_return error
+	err_return = nil
```

Comment:
@zzengcs you don't need  `err_return` here.

Reply:
it has been removed when addressing the previous two comments, tks